src/clj/kixi/hecuba/api/measurements.clj

```diff
@@ -34,6 +34,12 @@
 
 (def ^:private uploads-status-resource-path (p/resource-path-string :uploads-status-resource))
 
+(defn get-measurements-keys [sensor-keys session]
+  (let [sensor (sensors/get-by-type sensor-keys session)]
+    (if-let [alias-sensor (:alias-sensor sensor)]
```

Comment:
I don't get what's a sensor alias. Is it an integer?
So you either return this value or a map with device/sensor info.